src/test/CMakeLists.txt

```diff
@@ -1,15 +1,17 @@
 target_sources(test_main PUBLIC
+	CollisionTest.cpp
 	main.cpp
-	PhysicsTest.cpp
-	ScriptTest.cpp
-	ParticleTest.cpp
-	AssetTest.cpp
-	OptionalRefTest.cpp
-	RenderSystemTest.cpp
-	EventTest.cpp
-	ECSTest.cpp
-	SceneManagerTest.cpp
-	ValueBrokerTest.cpp
-	DBTest.cpp
-	Vector2Test.cpp
+	# PhysicsTest.cpp
+	# ScriptTest.cpp
+	# ParticleTest.cpp
+	# AssetTest.cpp
+	# OptionalRefTest.cpp
+	# RenderSystemTest.cpp
+	# EventTest.cpp
+	# ECSTest.cpp
+	# SceneManagerTest.cpp
+	# ValueBrokerTest.cpp
+	# DBTest.cpp
+	# Vector2Test.cpp
+	# Profiling.cpp
```

Comment:
Draai je dit uiteindelijk ook nog terug?

Reply:
ja maar voor testen is het wat makkelijker. Ik verander dit als ik geen feedback krijg dus net voor de PR merge